client/src/components/tags/CreateTagForm.jsx

```diff
@@ -27,17 +55,22 @@ export default function CreateTagForm()
                 </Label>
                 <Input 
                 name="Name"
-                placeholder="Enter a name for your tag"
+                value={tagName}
+                placeholder=""
                 onChange={(e) => {setTagName(e.target.value)}}/>
             </FormGroup>
             <FormGroup>
-                <Button type="submit" color="success" style={{float: "right"}}>
+              <ButtonToolbar  className="gap-2 "style={{float: "right"}}>
+              <Button type="submit" color="primary" style={{float: "right"}}>
                     Save
                 </Button>
+                <Button color="danger" onClick={() => {navigate("/tags")}}>Cancel</Button>
```

Comment:
Was it desired to have the "Cancel" button also render in the "Create a New Tag" view?